example_project_app/plugins.py

```diff
@@ -275,3 +275,26 @@ def perform_project_modify(
                     action=action.lower(),
                 ),
             )
+
+    def validate_form_app_settings(self, app_settings, project=None, user=None):
+        """Example implementation for app setting validation"""
+        errors = {}
+        if (
+            app_settings.get('scope')
```

Comment:
This example is nice, but there seems to be a mistake here. The method is supposed to receive all the app settings related to the app (plugin). each of those settings have their own scope, name etc, so this should be something like `app_settings[setting_name]['scope']` instead.